### DISCUSSION TITLE
Kindle Scribe -- auto warming seems to work fine -- but KOReader doesn't update warmness setting as it happens? & Auto light sensor adjusting?

### DISCUSSION THREAD

**Discussion by arooni** [koreader/koreader#11299] [General]
I *think* I read that this was expected behavior?  That there was no way for Koreader to grab the updated value for the warmth setting ?  Would be nice if it updated via the UI when it happened.

To be clear, I'm talking about setting the auto warmth setting from outside koreader on the normal kindle firmware where it auto moves from cold white light to warm light.  

Is the sensor for adjusting backlight based on the light sensor when enabled in the kindle stock firmware also working on the koreader experience?

**Comment by mergen3107** [comment]
> Koreader to grab the updated value for the warmth setting ?

There is no implementation yet, probably another listener is required.

> also working on the koreader experience

Yes, but Kindle firmware controls "brightness", while autowarmth controls "warmth".

In my Kindle Scribe, I just don't use it Kindle's auto brightness at all, I found it irritating by itself.
Auto warmth I find useful, it took me a few day to adjust all levels.

**Reply by mergen3107** [reply]
P.S. Kindle firmware also has scheduled warmth. Again, it won't be perfect because autowarmth from KOReader will keep bumping warmth from time to time (I think timer is around a minute or so, at least after manual warmth adjustment). 

So don't use both autowarmth methods :D
